src/Adapter/Driver/IbmDb2/Connection.php

```diff
@@ -156,6 +156,21 @@ public function disconnect()
         return $this;
     }
 
+     /**
+     * Disconnect a persistent resource
+     *
+     * @return ConnectionInterface
+     */
+    public function disconnectPersistent()
```

Comment:
Providing a completely different API for something that is an implementation detail is incorrect, and breaks LSP (expanding the interface for an adapter means that it is no longer an adapter)